picamera2/converters.py

```diff
@@ -1,8 +1,8 @@
 import numpy as np
 
-YUV2RGB_JPEG      = np.array([[1.0,   1.0,   1.0  ], [0.0, -0.344, 1.772], [1.402, -0.714, 0.0]])
+YUV2RGB_JPEG = np.array([[1.0, 1.0, 1.0], [0.0, -0.344, 1.772], [1.402, -0.714, 0.0]])
```

Comment:
Good question, the existing style checker didn't like the spacing too.  With flake8 though, I can add '# noqa' to make it allow this.  I could turn off the existing style checker now if you want, as I think flake8 should be stricter than the existing style checker, as it can catch things like imports that aren't needed etc. as far as I understand?